src/spec/reports_spec.js

```diff
@@ -211,7 +211,7 @@ describe('Reports page', () => {
     expect(reportsPage.isMaxRateDailyLimitDisplayed()).toEqual(true);
   });
 
-  it('should not show the daily limits for accounts that do not have limits', () => {
+  it('should show zero on the daily limits for accounts that do not have limits', () => {
```

Comment:
This fact is wrong: when the user _does not have limits_ he should not see the limits. He should see the limits **when there is a limit, even zero**.

Reply:
Oh ok, I' will update it. Thanks